server/routes/documents/show-cause/show-cause.controller.js

```diff
@@ -13,6 +13,7 @@
     return async function(req, res) {
       const { document } = req.params;
       const { jurorNumber } = req.query;
+      const { showCauseDate } = req.query;
```

Comment:
these can be merged into the same def:

`const { jurorNumber, showCauseDate } = req.query;`

Reply:
Done